webapp/src/ts/services/unread-records.service.ts

```diff
@@ -33,9 +36,13 @@ export class UnreadRecordsService implements OnDestroy {
   }
 
   private getRead() {
-    return this.dbService
-      .get({ meta: true })
-      .query('medic-user/read', { group: true });
+    return this.inited
+      .then(() => this.dbService.get({ meta: true }).query('medic-user/read', { group: true }))
+      .catch(err => {
+        if (err.status !== 404) {
+          throw err;
+        }
+      });
```

Comment:
Sometimes, this service tries to get unread records before the meta database is actually synced, and there is a ddoc to query,